File: problemas/iniciante/1179/1179.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let imprime = (nome, vetor) => {
    for (let i = 0; i < vetor.length; ++i) {
        console.log(`${nome}[${i}] = ${vetor[i]}`);
    }
};

let impar = [];
let par = [];
for (let i = 0; i < 15; ++i) {
    let valor = parseInt(lines.shift());

    if (Math.abs(valor) % 2) {
        impar.push(valor);

        if (impar.length === 5) {
            imprime("impar", impar);
            impar = [];
        }
    } else {
        par.push(valor);

        if (par.length === 5) {
            imprime("par", par);
            par = [];
        }
    }
}

imprime("impar", impar);
imprime("par", par);

File: problemas/iniciante/1179/1179.py
def imprime(nome, vetor):
    for i, valor in enumerate(vetor):
        print(f'{nome}[{i}] = {valor}')


impar = []
par = []
for _ in range(15):
    valor = int(input())

    if(abs(valor) % 2 == 1):
        impar.append(valor)

        if(len(impar) == 5):
            imprime("impar", impar)
            impar = []
    else:
        par.append(valor)

        if(len(par) == 5):
            imprime("par", par)
            par = []

imprime("impar", impar)
imprime("par", par)
